clients/web/htdocs/cdas/index.html

```diff
@@ -31,7 +31,7 @@
       	var debug = true;
         var map = new OpenLayers.Map('map');
         var variables = [  { url: "file://usr/local/web/WPCDAS/data/TestData.nc",  id: "t" },  ];
-        var django_server = 'http://localhost:8000/wps/';
+        var django_server = 'http://localhost:8080/wps/';
```

Comment:
@ThomasMaxwell we should try to move all "hard coded" path out into a "settings.py" file